### PR TITLE
feat(addon-doc): don't respect case-sensitive for tab link

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

image

## What is the new behavior?

image

### CODE REVIEW

**Comment by splincode** [comment]
@vladimirpotekhin @nsbarsukov hello, please review me

**Review by nsbarsukov — APPROVED** [review]

File: projects/addon-doc/tokens/url-lower-case.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import {tuiCreateToken} from '@taiga-ui/cdk';
+
+export const TUI_DOC_URL_LOWER_CASE = tuiCreateToken(false);
```
Comment (waterplea) [inline projects/addon-doc/tokens/url-lower-case.ts:3]: Looks overengineered to me. How about we just provide serializer in demo without all that configurations?

Comment (splincode) [reply projects/addon-doc/tokens/url-lower-case.ts]: fixed

Comment (waterplea) [reply projects/addon-doc/tokens/url-lower-case.ts]: I mean let's do it in demo app, because people might not want this behavior in addon-doc.

Comment (splincode) [reply projects/addon-doc/tokens/url-lower-case.ts]: But I wanna use it in tui-editor demo. Will I need to duplicate it?

**Pull request closed without merging**